base/reflection.jl

```diff
@@ -1050,32 +1051,70 @@ function MethodList(mt::Core.MethodTable)
     return MethodList(ms, mt)
 end
 
+# we are not actually returning the method table, but an array of methods
+# besides adding the documentation related to `:callable` I'm also rewording
+# Return the method table for `f`. ----> Return an array of methods for `f`.
```

Comment:
```suggestion
```
This is something worth noting in the PR but comments like these aren't typically added directly to the source code.

Reply:
Noted. Thanks.